src/CommandingOfficers/Tech.java

```diff
@@ -349,8 +361,23 @@ else if( !invalidDropCoords.contains(nmexy) )
         }
       }
 
+      // We only want to drop in near enemies; remove any destinations with no "enemy score".
+      Set<XYCoord> friendCoords = new HashSet<XYCoord>(friendScores.keySet());
+      for( XYCoord fc : friendCoords )
+        if( !enemyScores.keySet().contains(fc) )
+          friendScores.remove(fc);
+
+      // If that leaves us with no drop zones, then just reinforce the HQ.
```

Comment:
I suppose it's fair to note that this would require a second pass with `MassStrikeUtils`, but oh well.